package.json

```diff
@@ -116,6 +116,7 @@
     "markdown-it": "13.0.1",
     "module-alias": "2.2.2",
     "notion-to-md": "3.0.1",
+    "pako": "2.1.0",
```

Comment:
No need to install `pako`, you can use built-in `zlib` to achieve the same thing.

Reply:
Okay, I have made the modifications.